READEME.md

```diff
@@ -0,0 +1,15 @@
+# Calculator app
+
+App made for training skills.
+
+JavaScript, CSS.
+
+## Start App
+
+-App is ready to use and starting automatically.
+
+-To start you need Live Server.
+
+## instructions
```

Comment:
The same thing as in prev repo. Not making progress on headines ;p

Reply:
fixed